jobclient/java/src/main/java/com/twosigma/cook/jobclient/Disk.java

```diff
@@ -0,0 +1,104 @@
+package com.twosigma.cook.jobclient;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+public class Disk {
+    /**
+     *   request - disk space guaranteed for the job
+     *   limit - max disk space the job can use
+     *   type - type of disk for the job
+     */
+    private Double _request;
+    private Double _limit;
+    private String _type;
```

Comment:
shouldn't the type of this field be DiskType

Reply:
No, we transform the enum version of DiskType to the String representation of _type because that is what we ultimately feed into the Cook payload. Also, this allows us to not have to update the enum DiskType each time we add support for a new disk type. Added a clarifying comment

Reply:
I would get rid of the DiskType enum and just use string
@dposada thoughts?

Reply:
The enum seems like a nice convenience thing -- callers don't have to know or look up the valid string choices.